webextension/css/context.css

```diff
@@ -142,14 +142,16 @@ ul {
 }
 
 .pre {
-    /* font-family: Menlo,Monaco,Consolas,"Courier New",monospace; */
+    font-family: Menlo,Monaco,Consolas,"Courier New",monospace;
     display: block;
-    padding: 9.5px;
-    margin: 1em 0 5vh;
```

Comment:
The margin issue is because of removing it, we can retain/add it :-)

Reply:
If you will add here there is issue in overview context page !

Reply:
I have made the changes in latest changes, you can check !

Reply:
Yes, thank you !